src/standard/effectBuilder.html

```diff
@@ -193,6 +226,10 @@
       var m = expression.match(/([^\s]+?)\((.*)\)/);
       if (m) {
         var sig = { method: m[1], static: true };
+        if (this.properties[sig.method]) {
```

Comment:
To be compatible with `behaviors`, I think you can just change this to `this._propertyInfo[sig.method]`.  Could you add tests a `computed`-computed annotation function supplied by a behavior?